distribution_synthesis_inserts.fasta

```diff
@@ -2225,6 +2225,811 @@ tcaagaccacctacaaggccaagaagcccgtgcagatgcccggcgcctacaacgtcgacc
+TCGTCTGTATGGCAAACCGGATGCGCCGGAACTGTTTCTGAAACATGGCAAAGGCAGCGT
+GGCGAACGATGTGACCGATGAAATGGTGCGTCTGAACTGGCTGACCGAATTTATGCCGCT
+GCCGACCATTAAACATTTTATTCGCACCCCGGATGATGCGTGGCTGCTGACCACCGCGAT
+TCCGGGCAAAACCGCGTTTCAGGTGCTGGAAGAATATCCGGATAGCGGCGAGAACATTGT
+GGATGCGCTGGCCGTGTTTCTGCGTCGTCTGCATAGCATTCCGGTGTGCAACTGCCCGTT
+TAACAGCGATCGTGTGTTTCGTCTGGCCCAGGCGCAGAGCCGTATGAACAACGGCCTGGT
+GGATGCGAGCGATTTTGATGATGAACGTAACGGCTGGCCGGTGGAACAGGTGTGGAAAGA
+AATGCATAAACTGCTGCCGTTTAGCCCGGATAGCGTGGTGACCCACGGCGATTTTAGCCT
+GGATAACCTGATTTTCGATGAAGGCAAACTGATTGGCTGCATTGATGTGGGCCGTGTGGG
+CATTGCGGATCGTTATCAGGATCTGGCCATTCTGTGGAACTGCCTGGGCGAATTTAGCCC
+GAGCCTGCAAAAACGTCTGTTTCAGAAATATGGCATTGATAATCCGGATATGAACAAACT
+GCAATTTCATCTGATGCTGGATGAATTTTTCTAATAACTAGCATAACCCCTTGGGGCCTC
+TAAACGGGTCTTGAGGGGTTTTTTGACTAAGAGACCGTGCAC
+>Ecoli_SpecR
+GTGCACGGTCTCAGCAAGAACCTTGACCGAACGCAGCGGTGGTAACGGCGCAGTGGCGGT
+TTTCATGGCTTGTTATGACTGTTTTTTTGGGGTACAGTCTATGCCTCGGGCATCCAAGCA
+GCAAGCGCGTTACGCCGTGGGTCGATGTTTGATGTTATGGAGCAGCAACGATGTTACGCA
+GCAGGGCAGTCGCCCTAAAACAAAGTTAAACATCATGAGGGAAGCGGTGATCGCCGAAGT
+ATCGACTCAACTATCAGAGGTAGTTGGCGTCATCGAGCGCCATCTCGAACCGACGTTGCT
+GGCCGTACATTTGTACGGCTCCGCAGTGGATGGCGGCCTGAAGCCACACAGTGATATTGA
+TTTGCTGGTTACGGTGACCGTAAGGCTTGATGAAACAACGCGGCGAGCTTTGATCAACGA
+CCTTTTGGAAACTTCGGCTTCCCCTGGAGAGAGCGAGATTCTCCGCGCTGTAGAAGTCAC
+CATTGTTGTGCACGACGACATCATTCCGTGGCGTTATCCAGCTAAGCGCGAACTGCAATT
+TGGAGAATGGCAGCGCAATGACATTCTTGCAGGTATCTTCGAGCCAGCCACGATCGACAT
+TGATCTGGCTATCTTGCTGACAAAAGCAAGAGAACATAGCGTTGCCTTGGTAGGTCCAGC
+GGCGGAGGAACTCTTTGATCCGGTTCCTGAACAGGATCTATTTGAGGCGCTAAATGAAAC
+CTTAACGCTATGGAACTCGCCGCCCGACTGGGCTGGCGATGAGCGAAATGTAGTGCTTAC
+GTTGTCCCGCATTTGGTACAGCGCAGTAACCGGCAAAATCGCGCCGAAGGATGTCGCTGC
+CGACTGGGCAATGGAGCGCCTGCCGGCCCAGTATCAGCCCGTCATACTTGAAGCTAGACA
+GGCTTATCTTGGACAAGAAGAAGATCGCTTGGCCTCGCGCGCAGATCAGTTGGAAGAATT
+TGTCCACTACGTGAAAGGCGAGATCACCAAGGTAGTCGGCAAATAAGACTAAGAGACCGT
+GCAC
+>3_APOSTROPHE_Identity_Linker
+GGTCTCATACTATGTCTTCGCCTACGGATTGTCTGTCAAGGCATGAGACC
+>3_APOSTROPHE_Identity_Linker_2
+GGTCTCATACTTGAGACCGGAGCCTACGGATTGTCTGTCAAGGCATGAGACC
+>3_APOSTROPHE_Identity_Linker_3
+GGTCTCAAATGATGTCTTCGCCTACGGATTGTCTGTCAAGGCATGAGACC
+>3_APOSTROPHE_Identity_Linker_4
+GGTCTCAAATGTGAGACCGGAGCCTACGGATTGTCTGTCAAGGCATGAGACC
+>5_APOSTROPHE_Identity_Linker
+GGTCTCAGACCAAACACGTGGCAAACATTCCGAAGACATGGAGTGAGACC
+>5_APOSTROPHE_Identity_Linker_2
+GGTCTCAGACCAAACACGTGGCAAACATTCCACCGGTCTCAGGAGTGAGACC
+>5_APOSTROPHE_Identity_Linker_3
+GGTCTCAGACCAAACACGTGGCAAACATTCCAGAAGACATGCTTTGAGACC
+>5_APOSTROPHE_Identity_Linker_4
+GGTCTCAGACCAAACACGTGGCAAACATTCCACCGGTCTCAGCTTTGAGACC
+>T1_Transcriptional_Terminator
+GTGCACGGTCTCTGCTTCTGTAACAGAGCATTAGCGCAAGGTGATTTTTGTCTTCTTGCG
+CTAATTTTTTCGCTTGAGACCGTGCAC
+>Lox71_select_Lox66
+GTGCACGGTCTCAAAGGTACCGTTCGTATAATGTATGCTATACGAAGTTATCTTGAGGAC
+TTTTGCGTTCGAAGGTGAGACGACCGGTGAATTCCAATTGCGTCTCAATGACGATAGAAC
+GACCAGGTAGCATAACTTCGTATAATGTATGCTATACGAACGGTAATGATGAGACCGTGC
+AC
+>3_APOSTROPHE_Part_Switch_Linker
+GGTCTCATACTTGTGATGTCTTCGCCTACGGATTGTCTGTCAAGGCATGAGACC
```

Comment:
These short parts cannot be built by Twist, whose minimum synthesis length is 300bp. They need to have padding and flanking sequences added to them. See the Anderson Promoters package for an example of how this has been done.

Reply:
By my count, we have 36 parts under 300bp in length. This is just to note my intention to pad all of those parts.

Reply:
Yes, I added ~50%GC hair-pin free random sequence to all of my OYC parts that were under 300bp. I made those parts 310bp total.